Mathlib/Order/Atoms.lean

```diff
@@ -1111,10 +1117,10 @@ end IsModularLattice
 
 namespace «Prop»
 
-@[simp] theorem isAtom_iff {p : Prop} : IsAtom p ↔ p := by
+theorem isAtom_iff {p : Prop} : IsAtom p ↔ p := by
   simp [IsAtom, show ⊥ = False from rfl, fun q r : Prop => show q < r ↔ _ ∧ _ from .rfl]
```

Comment:
This proofis now much shorter with `isAtom_iff_eq_top`, right? Or is the instance needed not present yet in this file?

Reply:
Well spotted; just need to move the `instance : IsSimpleOrder Prop` up.